assets/pages/support-landing/supportLanding.jsx

```diff
@@ -44,17 +49,19 @@ const content = (
         headings={['Help us deliver', 'the independent', 'journalism the', 'world needs']}
         highlights={['Support', 'The Guardian']}
       />
-      <Contribute
-        copy="Your contribution funds and supports The Guardian's journalism."
-      >
-        <ContributionSelectionContainer />
-        <ContributionPaymentCtasContainer
-          PayPalButton={PayPalContributionButtonContainer}
-        />
-      </Contribute>
-      <ThreeSubscriptions />
+      <section id={supporterSectionId}>
+        <Contribute
+          copy="Your contribution funds and supports The Guardian's journalism."
+        >
+          <ContributionSelectionContainer />
+          <ContributionPaymentCtasContainer
+            PayPalButton={PayPalContributionButtonContainer}
```

Comment:
Why the ContributionPaymentCtasContainer  receives a PayPalButton 
 but no StripeButton?

Reply:
That didn't change in this PR, that's an indentation change. However, that's the case because there's no Stripe button on this page.